client/editSchedule.js

```diff
@@ -1,37 +1,27 @@
-var editingRoundReact = new ReactiveVar(null);
+var eventToEditReact = new ReactiveVar(null);  // TODO I don't think this needs to be reactive - Lars
```

Comment:
You've got it. More specifically: Line 12: `eventToEditReact.get()` is run as part of blaze rendering the template. If you dig into that `.get()` method, you'll see that there is code that "wires up" the reactive dependency that ensures whenever `eventToEditReact.set()` is called, the blaze rerender code runs. This is possible because Blaze does its rendering inside of a call to [`Tracker.autorun`](http://docs.meteor.com/#/full/tracker_autorun), which I believe stashes a function pointer somewhere where `eventToEditReact.get()` is able to get to it.